completion/bash/toolbox

```diff
@@ -52,7 +52,7 @@ __toolbox() {
   esac
 
   local command
-  if [ "${COMP_WORDS[1]}" = --verbose ]; then
+  if [ "${COMP_WORDS[1]}" = --verbose || "${COMP_WORDS[1]}" = --very-verbose ]; then
```

Comment:
It should be `[ a ] || [ b ]`, not `[ a || b ]`.

We should enable`shellcheck` for this file. It would have triggered [SC2109](https://github.com/koalaman/shellcheck/wiki/SC2109).

Sorry for not noticing this earlier.